dvc/stage.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, fname):
 
 class StageFileAlreadyExistsError(DvcException):
     def __init__(self, relpath):
-        msg = "stage '{}' already exists".format(relpath)
+        msg = "not overwriting '{}'".format(relpath)
```

Comment:
For the record: this is the only place where this exc is used, so we can reword it like this.